.github/workflows/ci.yml

```
@@ -49,14 +84,15 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features --all
+          args: ${{ maxrix.features }} --workspace
```

This line still has a typo.
```suggestion
          args: ${{ matrix.features }} --workspace
```

---

Wow I _really_ cannot spell `matrix`, can I?